src/HtmlQuotations.ts

```diff
@@ -230,65 +239,19 @@ export function cutById(document: Document): boolean {
  * Cust the last non-nested blockquote with wrapping elements.
  *
  * @param {Document} document - The document to cut the element from.
+ * @param {cutQuoteOption} options - Extra options.
  * @return {boolean} Whether a corresponding quote was found or not.
  */
-export function cutBlockquote(document: Document): boolean {
+export function cutBlockquote(document: Document, options?: cutQuoteOption): boolean {
   const quote = <Node>XPath.select(
     "(.//*[local-name(.)='blockquote'])" +
     "[not(@class=\"gmail_quote\") and not(ancestor::blockquote)]" +
     "[last()]"
   , document, true);
 
-  if (!quote)
+  if (!quote || shouldNotRemoveQuote(quote, options))
     return false;
 
   quote.parentNode.removeChild(quote);
   return true;
-};
-
-/**
- * Cuts div tag that wraps a block starting with "From:".
- *
- * @param {Document} document - The document to cut the element from.
- * @return {boolean} Whether a corresponding quote was found or not.
- */
-export function cutFromBlock(document: Document): boolean {
```

Comment:
Was that just not used anywhere anymore?

Reply:
Looks like it was remove in thie PR : https://github.com/quentez/talonjs/commit/191d79d26c43eac297abd19fbdef796e37ece3f8